www/js/lib/settingsStore.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-define([], function () {
+define(['jquery', 'uiUtil'], function ($, uiUtil) {
```

Comment:
Remove 'uiUtil' from here, and also fromt he function. Not needed as you are requiring it below. This will also get rid of the codefactor warning.

Reply:
@gaurav7019 It also looks like you can remove jquery and $, if you're not using jquery at all in settingsStore. We want to move away from jQuery, so best to remove unless explicitly needed.

Reply:
Done!